portfolio/src/main/webapp/comments.js

```diff
@@ -55,6 +55,19 @@ class Comments {
     commentElement.innerText = text;
     return commentElement;
   }
+
+  /**
+   * Returns an error if the response status is not between 200 and 299, i.e not OK.
+   * @param {*} response - The HTTP response received from the servlet.
+   * @return {*} - If an error is not thrown, returns the json response.
+   */
+  checkFetchError(response) {
+    if (response.status >= 200 && response.status <= 299) {
```

Comment:
Please review the API for a response object. This can be replaced with response.ok()

https://developer.mozilla.org/en-US/docs/Web/API/Response